util/all_tests.json

```diff
@@ -96,14 +96,17 @@
     "cmd": ["ssl/ssl_test"]
   },
   {
-    "cmd": ["crypto/mem_test"]
+    "cmd": ["crypto/mem_test"],
+    "skip_sde": true
```

Comment:
Can we fix this later? Do we have a tracking issue?

Reply:
I had been experimenting skipping this in Codebuild since the tests had been taking quite a while. Seems like they don't take much more time on a platform with sufficient cores, so I've added them back :).